### PR TITLE
update generate_rust.sh to download protobuf-codegen.

### PR DESCRIPTION
**Pull request by hicqu**
Previously we download `protobuf` to generate rust code. However the program's name has changed to `protobuf-codegen`.

### CODE REVIEW

**Comment by hicqu** [comment]
ping @BusyJay @overvenus PTAL thanks.

**Comment by overvenus** [comment]
Does it still require `protoc`?

**Comment by hicqu** [comment]
It still requires protoc.

**Comment by siddontang** [comment]
LGTM

**Comment by hicqu** [comment]
ping @BusyJay @overvenus .

File: generate_rust.sh [hunk 1]
```diff
@@ -6,10 +6,10 @@ if ! check_protoc_version; then
 	exit 1
 fi
 
-# install rust-protobuf if it's missing
-if ! cmd_exists protoc-gen-rust; then
-    echo "missing rust-protobuf, try to download/install it"
-    cargo install protobuf || exit 1
+# install protobuf-codegen if it's missing
+if ! cargo install --list|grep "protobuf-codegen"; then
```
Comment (Hoverbear) [inline generate_rust.sh]: Should we check the version?

File: generate_rust.sh [hunk 2]
```diff
@@ -6,10 +6,10 @@ if ! check_protoc_version; then
...
+    echo "missing protobuf-codegen, try to download/install it"
```
Comment (Hoverbear) [inline generate_rust.sh]: `trying to download/install it`

**Comment by hicqu** [comment]
ping @Hoverbear PTAL thanks.

**Pull request merged**